data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,370 @@
+			choice
+				`	"Tell me more."`
+					goto tellme
+				`	"How do you know me?"`
+					goto know
+				`	(Run away).`
+			`	They seem surprised at your sudden change of mind, but make no effort to stop you. You proceed to take the antigrav back to the shipyard without any incidents.`
+				decline
+			label know
+			`	They grin, "Us knowing you is a necessity for you to land on our homeworld."`
+			label tellme
+			`	"We have a very interesting proposition for you: should you agree to bring supplies to our brethren, the Unfettered Hai, we will offer you a ship, for a modest price of 6.5 million credits. This is a very useful ship, and you should be able to repay this quite easily by running more of the same kind of jobs. It is indeed appropriate for bringing this important food cargo safely to destination, and even defend yourself, if need be. You see, these supplies are needed on short notice, on <destination>, and we lack the time and personnel to bring them."`
+			`	They take you into the cave, finally showing the ship they were referring to. "This is a mighty relic of the past, from the times our Empire used to control the stars; a Ladybug!`
+			scene "thumbnail/hai ladybug"
+			`	"Of course, it has some... drawbacks, compared to its golden time, and don't expect the best price in a shipyard either. Nevertheless it is a fine vessel, capable of safely taking a respectable amount of cargo across large distances! So, what do you say, <last>? This is the opportunity of a lifetime, as they simply don't make ships like these anymore!"`
+			choice
+				`	"I will take this deal."`
+					goto take
+				`	"I am not interest in this deal."`
+			`	"A shame, this probably means we will not meet again," they say, before leaving, as discreetly as they came.`
+				decline
+			label take
+			action
+				give ship "Ancient Ladybug" "Good Omen"
+				fine 6500000
+				log "Accepted a somewhat shady deal for an old Hai ship, a Ladybug, that seems to still be in good functioning state."
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go trough official means for this, we will have to present it as a fine. Worry not, we will make it something of no consequence, like an old Korath forbidden tech you would not have known about, that we had to confiscate."`
+			`	You quickly realize they are not asking you for your opinion at all. In fact, you notice the fine has already been issued before you could say anything. "Oh, and you should not change its colors, you'll understand why later."`
+				accept
+
+# This mission should trigger right after the one above.
```

Comment:
I had to take off and land again to pick this one up.

Reply:
Damn, I did some to accept stuff hoping u wouldnt
I'll instead make it happen as you exit the shipyard ig